packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -644,7 +649,8 @@ function hideOrUnhideAllChildren(finishedWork, isHidden) {
         }
       } else if (
         node.tag === SuspenseComponent &&
-        node.memoizedState !== null
+        node.memoizedState !== null &&
+        node.memoizedState.dehydrated === null
```

Comment:
What is this about? I don't see you setting it in this PR.

Is this a behavior change? Wasn't the feature logging-only? Or was this always `null` anyway?

Reply:
This probably should've been in #16346 but it was not observable because we weren't scheduling an Update effect in this scenario. Now that we do, we need to differentiate what purpose the Update effect has.